pycbc/events/trigger_fits.py

```diff
@@ -45,23 +48,48 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
 # Public License for more details.
 
+import logging
 import numpy
 from scipy.stats import kstest
 
+def exponential_fitalpha(vals, thresh, w):
+    """
+    Maximum likelihood estimator for the fit factor for
+    an exponential decrease model
+    """
+    return 1. / (numpy.average(vals, weights=w) - thresh)
+
+
+def rayleigh_fitalpha(vals, thresh, w):
+    """
+    Maximum likelihood estimator for the fit factor for
+    a rayleigh distribution of events
+    """
+    return 2. / (numpy.average(vals**2., weights=w) - thresh**2.)
```

Comment:
probably a good idea to take advantage of this rewrite to PEP8-ize the algebra here, ie spaces around '**'

Reply:
(and 'rayleigh' -> 'Rayleigh')